src/routes/v1/userRoutes.js

```diff
@@ -1,13 +1,21 @@
 const express = require('express');
 const userController = require('../../controllers/userController');
-const {
-  userSignUpValidation,
-  userSignInValidation,
-} = require('../../middlewares/errorHandling');
+const { userSignUpSchema, userSignInSchema } = require('../../models/user');
+const { commonValidationHandler } = require('../../middlewares/errorHandling');
 const userRouter = express.Router();
 
-userRouter.post('/register', userSignUpValidation, userController.signUp);
+userRouter.post(
+  '/register',
+  commonValidationHandler(userSignUpSchema),
+  // eslint-disable-next-line comma-dangle
+  userController.signUp
```

Comment:
deconstruct signUp and signIn from userController

Reply:
resolved with deconstructing signup and signin